src/utils/date.ts

```diff
@@ -212,3 +216,14 @@ export function getDaysDiff(date1: Date, date2: Date): number {
   const ts2 = date2.getTime();
   return (ts1 - ts2) / (1000 * 3600 * 24);
 }
+
+/**
+ * Set time of the day to the end.
+ *
+ * @param {Date} date Date.
+ * @returns {Date} Date with time set to end of day .
+ */
+export function setEndOfDay(date: Date): Date {
```

Comment:
<img src="https://media3.giphy.com/media/OEyVshb3z2mnm/giphy.webp?cid=ecf05e47aw57i9opi3nk90vsviuqvsrv8uaoz7sb3nwtq3zo&rid=giphy.webp&ct=g" />

THE END OF DAYS